_commands/basics/sort.md

```diff
@@ -3,22 +3,59 @@
 
 sort
 --
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
+"Sort" is a standard command that sorts and outputs the input it was given.  
+Blank space is the default field separator.
 
 <!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+$ sort names
+  Aaron
+  Brian
 ~~~
 
 <!--more-->
 
 ### Useful Options / Examples
 
-#### Example command
+The -n flag lets you sort numerical input.
 
-##### Break it down
+~~~ bash
+$ sort -n numbers
+  1
+  5
+  9
+  10
+  17
+~~~
+
+The -k flag lets you choose the column by which your input gets sorted.
```

Comment:
format all flags and any other inline code as code, write `-k` instead of -k (surround it with \`)